cases-SEP31/send.test.js

```diff
@@ -38,7 +38,10 @@ describe("POST /send", () => {
   });
 
   it("fails with no amount", async () => {
-    const headers = { Authorization: `Bearer ${jwt}` };
+    const headers = {
+      Authorization: `Bearer ${jwt}`,
+      "Content-Type": "application/json",
+    };
```

Comment:
`multipart/form-data` is the default content type. [SEP-24](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#request) and SEP-6 use it.

Reply:
We could add it as an accepted type here too, but I believe we decided against adding `application/json` to SEP24 or SEP6 because we didn't want to require the anchor to support both encodings.